src/components/cms/SkillCreator/SkillWizard.js

```diff
@@ -1045,6 +1065,38 @@ class SkillWizard extends Component {
             />
             <CenterDiv>
               <DropDownDiv>
+                <Popover
+                  open={open}
+                  anchorEl={anchorEl}
+                  onClose={this.handleClose}
+                  anchorOrigin={{
+                    vertical: 'bottom',
+                    horizontal: 'center',
+                  }}
+                  transformOrigin={{
+                    vertical: 'top',
+                    horizontal: 'center',
+                  }}
+                >
+                  <Text>
+                    Open Source Skills will show in the public directory and are
+                    editable by everyone just like a Wikipedia article. Private
+                    skills are only editable by the creator/owner of the skill.
+                    They are not listed in the public directory and can
+                    currently only be used by the creator/owner.
+                  </Text>
+                </Popover>
+                <DetailText>
+                  Open Source and Public Skill(Coming Soon) (
+                  <span
+                    onClick={this.handleClick}
+                    style={{ color: '#4285F4', cursor: 'pointer' }}
+                  >
+                    ?
+                  </span>
+                  ):
+                </DetailText>{' '}
+                <Switch color="primary" checked={true} disabled={true} />
```

Comment:
Add functionality to toggle. Toggle between Private and Public. Check issue.

Reply:
@akshatnitd The issue says when switched off show private skill. Does it mean to switch to the bot builder page ? If so then we don't need to disable the button.

Reply:
@harsh-jindal It means, then slider is on right side, the text should read Public/Open source skills, when it is on left side, the text should read Private

Reply:
@akshatnitd I understood the issue but that's not a good idea because it creates confusion. When I turn of the slider if the text changes to private skills it gives a effect like `private skill:` is off. So the user clicks on the button and he see `open source and public skill` is on. IMO text shall not change on toggling of the slider to avoid the users getting confused.